pkg/process/net/common.go

```diff
@@ -476,6 +476,31 @@ func (r *RemoteSysProbeUtil) GetConnTrackHost() ([]byte, error) {
 	return data, nil
 }
 
+// GetBTFLoaderInfo queries btf_loader_info to get information about where btf files came from
+func (r *RemoteSysProbeUtil) GetBTFLoaderInfo() ([]byte, error) {
```

Comment:
I think this can live in `common_linux.go` since it won't ever be called from windows.

Reply:
This exists to satisfy the `SysProbeUtil` interface - can I modify `SysProbeUtil` to be platform specific, i.e. it wouldn't have `GetBTFLoaderInfo` on windows? Or does that not make sense? Currently on every platform, linux windows etc, it is implemented by the same struct `RemoteSysProbeUtil` as far as I know

Reply:
Can you implement a stub for windows then?